glue/src/main/scala/com/gu/tableversions/glue/GlueMetastore.scala

```diff
@@ -0,0 +1,184 @@
+package com.gu.tableversions.glue
+
+import java.net.URI
+
+import cats.effect.Sync
+import cats.implicits._
+import com.amazonaws.services.glue.AWSGlue
+import com.amazonaws.services.glue.model.{
+  CreatePartitionRequest,
+  DeletePartitionRequest,
+  GetPartitionsRequest,
+  GetPartitionsResult,
+  GetTableRequest,
+  PartitionInput,
+  StorageDescriptor,
+  TableInput,
+  UpdatePartitionRequest,
+  UpdateTableRequest,
+  Partition => GluePartition,
+  Table => GlueTable
+}
```

Comment:
Is it not possible to do an `import com.amazonaws.services.glue.model._` as you're aliasing some of the imports?

Reply:
i think you can put _ at the end of the import list. 
But there's some other classes in that glue.model package that have the same name as our own model

Reply:
I see. Makes sense as it is then!